heltour/tournament/models.py

```diff
@@ -890,10 +896,11 @@ def agreed_to_tos(self):
             date_first_agreed_to_tos=now
         )
 
-
     def get_access_token(self):
         return self.oauth_token.access_token
 
+    def token_valid(self):
+        return self.oauth_token.access_token is not None and not self.oauth_token.is_expired()
```

Comment:
Use the `get_access_token` function above, or, better yet, add a function to `oauth_token` which does this full check